src/main/sensors/esc_sensor.c

```diff
@@ -135,45 +136,45 @@ bool escTelemetryInit(void)
     portOptions_t options = (SERIAL_NOT_INVERTED);
 
     // Initialize serial port
-    escTelemetryPort = openSerialPort(portConfig->identifier, FUNCTION_TELEMETRY_ESC, escTelemetryDataReceive, ESC_TLM_BAUDRATE, MODE_RX, options);
+    escSensorPort = openSerialPort(portConfig->identifier, FUNCTION_TELEMETRY_ESC, escSensorDataReceive, ESC_SENSOR_BAUDRATE, MODE_RX, options);
```

Comment:
@mikeller good to know you changed that already. Had planned to look into the configurator tonight.